### DISCUSSION TITLE
warning message requested RAM and Bandwidth no mount is available

### DISCUSSION THREAD

**Discussion by HilmyThoriq** [manoelcampos/cloudsim-plus#443] [Q&A] [answered]
![screenshottt](https://github.com/cloudsimplus/cloudsimplus/assets/81411468/c8584030-af78-4363-bbbf-2744611770c5)

Hello!

I want to ask about this warning how to prevent this kind of error ? does i need to increase my host RAM and Bandwidth? and also does this warning message does affect the execution time of cloudlet in each ?

Thanks

**Comment by manoelcampos** [comment, marked as answer]
You have to avoid using UtilizationModelFull for ram when you have multiple Cloudlets inside the same Vm. Check UtilizationModelDynamic or UtilizationModelStochastic

**Reply by HilmyThoriq** [reply]
Iam already try using UtilizationModelDynamic or Stochastic but still got the same problem even though some of the RAM and Bandwidth this time is allocated does there's another way to prevent it?

**Reply by manoelcampos** [reply]
No matter which kind of UtilizationModel you use. If you have two Cloudlets running in parallel (using a CloudletSchedulerTimeShared) into the same VM and each Cloulet requires 60% of RAM, both Cloudlets would need 120% of RAM, which is not possible. This way, one Cloudlet will start using Virtual Memory (which delays Cloudlet execution). That is why you get warnings. Therefore, even using some UtilizationModel other than UtilizationModelFull, you have to ensure that it will not request more resources for a single Cloudlet than there is available.
